conans/model/conan_generator.py

```diff
@@ -41,3 +41,6 @@ def content(self):
     @abstractproperty
     def filename(self):
         raise NotImplementedError()
+
+    def sorted_components(self, cpp_info):
```

Comment:
We have to agree on the interface of this method in order to document it

Reply:
Agreed to postpone the documentation of this method to use it in other generators